src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java

```diff
@@ -73,7 +73,7 @@
      * selected asset numbers, the system control amount is redistributed equally among the assets
      * when the distribution method is "distribute cost equally".
      *
-     * @see org.kuali.kfs.kns.web.struts.action.KualiAction#refresh(org.apache.struts.action.ActionMapping,
+     * @see org.kuali.rice.kns.web.struts.action.KualiAction#refresh(org.apache.struts.action.ActionMapping,
```

Comment:
I wonder why there is a change to the rice version here. It seems that this change was part of the license update, I wonder if it was accidental? I don't have access to the KualiCo financials project so I cannot check out what this code is there.